.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug report
+about: Create a report to help us improve
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1.
+2.
+3.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Technical details:**
+ - OS:
+ - Rust Version:
+ - Cargo Version:
+ - Release Channel:
```

Comment:
This is info which ideally we've have a single-shot command to automatically fill this rather than asking the user to explicitly write this in. It may mean that we need to update `cargo version --verbose` perhaps, or maybe even add `cargo report-bug` to print this sort of information.

Reply:
Either sound good to me.